pvt/benchmarks/multihop.ts

```diff
@@ -39,6 +39,11 @@ async function main() {
   await multihop((index: number) => getWeightedPool(vault, tokens, 38, index), false);
   await multihop((index: number) => getWeightedPool(vault, tokens, 38, index), true);
```

Comment:
Do we want to keep the benchmark at 38?

Reply:
It does 50 right after that.

Reply:
Good catch on the comment! Will fix that (and the calculation).